frontend/src/components/Table/Table.scss

```diff
@@ -37,48 +37,31 @@
     background-color: #eaeaea;
   }
 
-  &__row {
-    &_withHover {
-      &:hover {
-        background-color: $color-light-green;
-      }
-    }
-  }
-
   &__cell {
+    max-height: 50 * $px;
     padding: (16 * $px) 0 (16 * $px) $cell-indent;
+    font-size: 14 * $px;
+    font-weight: 400;
+    line-height: 18 * $px;
```

Comment:
like you?

Reply:
yes and no, now it seems that 18px follows from 16*2 paddings, but I had some troubles in this place, so I want it to be explicitly specified in case if we change paddings and the row height.